src/network/networkd-network.c

```diff
@@ -749,10 +749,7 @@ int config_parse_tunnel(const char *unit,
                 return 0;
         }
 
-        if (netdev->kind != NETDEV_KIND_IPIP &&
-            netdev->kind != NETDEV_KIND_SIT &&
-            netdev->kind != NETDEV_KIND_GRE &&
-            netdev->kind != NETDEV_KIND_GRETAP &&
+        if (!IN_SET(netdev->kind, NETDEV_KIND_IPIP, NETDEV_KIND_SIT, NETDEV_KIND_GRE, NETDEV_KIND_GRETAP) &&
```

Comment:
This line isn't matched completely, maybe edit it by hand.

Reply:
I'll expand the cocci expression to match such cases and rerun it on the file.